CoreScriptsRoot/Modules/Settings/Pages/LeaveGame.lua

```diff
@@ -80,9 +86,9 @@ local function Initialize()
 		buttonSize = UDim2.new(0, 300, 0, 80)
 	end
 
-	this.LeaveGameButton = utility:MakeStyledButton("LeaveGame", "Leave", buttonSize)
+	this.LeaveGameButton = utility:MakeStyledButton("LeaveGame", "Leave", buttonSize, this.LeaveFunc)
 	this.LeaveGameButton.NextSelectionRight = nil
-	this.LeaveGameButton:SetVerb("Exit")
+	---this.LeaveGameButton:SetVerb("Exit")
```

Comment:
can we flag this change? Everything should work ok, but its a pretty big change and I want to be able to roll back quickly if something goes wrong. Flag is ok to default to true though I think